src/vs/workbench/api/electron-browser/mainThreadWebview.ts

```diff
@@ -200,9 +200,9 @@ export class MainThreadWebviews extends Disposable implements MainThreadWebviews
 			return;
 		}
 
-		const targetGroup = this._editorGroupService.getGroup(viewColumnToEditorGroup(this._editorGroupService, showOptions.viewColumn));
+		const targetGroup = this._editorGroupService.getGroup(viewColumnToEditorGroup(this._editorGroupService, showOptions.viewColumn)) || this._editorGroupService.getGroup(webview.group || 0);
 		if (targetGroup) {
-			this._webviewService.revealWebview(webview, targetGroup || this._editorGroupService.getGroup(webview.group || ACTIVE_GROUP), !!showOptions.preserveFocus);
```

Comment:
`webview.group || ACTIVE_GROUP` fails for the web view in first view column as `web view.group` is `0` and `0 || -1 === -1`. See https://github.com/Microsoft/vscode/commit/2e6db8d955046349a941ab814183cf480b0e274b